client.go

```diff
@@ -1956,6 +1961,9 @@ func (c *ToznySDKV3) ListSecrets(ctx context.Context, options ListSecretsOptions
 				if err != nil {
 					return nil, err
 				}
+				if len(identities.SearchedIdentitiesInformation) < 1 {
+					return nil, fmt.Errorf("ListSecrets: no identity found with clientID %s", record.Metadata.WriterID)
```

Comment:
This doesn't feel like the correct behavior at the moment. If a user that I shared a secret with is deleted (not sure what the state of cleanup is for that but regardless, state issues can occur) listing secrets should be resilient and return as much as it can. We didn't catch this in a previous review but I think most of this for loop should best efforts return as much data as it can. Ignoring errors or finding another way to report them. It feels like a really bad user experience to have a list method return nothing because one item had a problem. May not be highest priority to resolve now though